config/examples/copyoffload-containerprofile.yaml

```diff
@@ -11,24 +11,38 @@ data:
     - name: DW_GLOBAL_lus
       optional: false
   numPorts: 1
-  spec:
-    containers:
-      - name: copy-offload-srvr
-        image: nnf-dm-copy-offload:latest
-        command:
-          - /nnf-copy-offload
-          - --cert
-          - $(TLS_CERT_PATH)
-          - --cakey
-          - $(TLS_KEY_PATH)
-          - --tokenkey
-          - $(TOKEN_KEY_PATH)
-          - --addr
-          - :$(NNF_CONTAINER_PORTS)
-    # This ServiceAccount is used by the nnf-dm-copy-offload server to talk to
-    # the Kubernetes API server. This ServiceAccount and its Role are
-    # defined in config/copy-offload.
-    # Normal user containers should not require Kubernetes API server privilege
-    # and should not have a service account.
-    serviceAccountName: nnf-dm-copy-offload
   retryLimit: 2
+  # Copy offload needs to use mpiSpec to use the mpi-operator model
+  # TODO: Add a webhook check that any template with a serviceAccountName == nnf-dm-copy-offload
+  # needs to have an mpiSpec
+  mpiSpec:
+    runPolicy:
+      cleanPodPolicy: Running
+    mpiReplicaSpecs:
+      Launcher:
+        template:
+          spec:
+            containers:
+              - name: copy-offload-srvr
+                image: ghcr.io/nearnodeflash/nnf-dm-copy-offload:latest
```

Comment:
Put this back to "nnf-dm-copy-offload:latest".  The `config/begin-examples` in nnf-dm will expand it.

And for the next `image:` line the Worker.